src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,7 @@
+databaseChangeLog:
+  - include:
+      file: /db/changelog/changes/01-create-books-table.yml
+  - include:
+      file: /db/changelog/changes/02-add-column-is-deleted-books-table.yml
+  - include:
+      file: /db/changelog/changes/03-add-not-null-constraint-to-is-deleted-books-table.yml
```

Comment:
I do not think we should add the `is_deleted` and its constraint as separate files, all of it could be just part of the books table creation script because they are all in a single PR and even in a single table so why not

Reply:
This is the historical result of my changes to the database. Splitting into multiple files was unintentional.

Reply:
I think I understood. I could make a rollback